zircon.core/src/commonTest/kotlin/org/hexworks/zircon/CharacterTilesAsString.kt

```diff
@@ -0,0 +1,34 @@
+package org.hexworks.zircon
+
+import org.hexworks.zircon.api.data.CharacterTile
+import org.hexworks.zircon.api.data.Position
+import org.hexworks.zircon.api.data.Size
+import org.hexworks.zircon.api.data.Tile
+import org.hexworks.zircon.api.graphics.TileComposite
+
+/**
+ * Converts a collection of character tiles (usually found at [TileComposite.tiles]) into a string.
+ *
+ * Note that this is just complicated enough to warrant its own unit tests (in jvmTest).
+ */
+fun TileComposite.characterTilesAsString(padToSize: Size = Size.zero()): String {
```

Comment:
This is already present in your other PR. I'm gonna leave it as-is but I'd suggest rebasing to `master` after this one gets merged so there will be no conflicts.